src/__init__.py

```diff
@@ -5,30 +5,42 @@
 from dotenv import load_dotenv
 
 from src.models import db
-from src.user import usr
-from src.config.config import TestingConfig
+from src.google import google_bp
+from src.home import home
+from src.manage import create_tables
 
 load_dotenv()
 
 def create_app(test_config=None):
     app: Flask = Flask(__name__, instance_relative_config=True)
     
     if test_config is None:
+        
+        # Heroku Postgresql hack.
+        db_url = str(os.environ.get('DATABASE_URL'))
+        if db_url.startswith('postgres://'):
+            db_url = db_url.replace('postgres://', 'postgresql://')
```

Comment:
Err why this though? Is it not better to just put the right `DATABASE_URL` as expected in the env variable instead of changing the string here?

Reply:
It's readonly on heroku